prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -79,15 +79,27 @@ public class PharmanetValidationRule : AutomaticAdjudicationRule
 
         public override async Task<bool> ProcessRule(Enrollee enrollee)
         {
-            if (enrollee.Certifications == null || !enrollee.Certifications.Any())
+            var certifications = enrollee.Certifications.Where(c => c.License.Validate);
+
+            // If enrollee choses device provider, add device provider as a licence and check it against PharmaNet
+            if (enrollee.HasCareSetting(CareSettingType.DeviceProvider) || enrollee.DeviceProviderIdentifier != null)
```

Comment:
If, somehow, a user could have the care setting DeviceProvider, but a null Identifier, we wouldn't want to try to find a cert with `null` for the License Number. I think this can just be 
```c#
if (!string.IsNullOrWhitespaceenrollee.DeviceProviderIdentifier))
```

Reply:
Makes sense, removed the other check